aeon/analysis/block_plotting.py

```diff
@@ -29,11 +29,15 @@
 def gen_hex_grad(hex_col, vals, min_l=0.3):
     """Generates an array of hex color values based on a gradient defined by unit-normalized values."""
     # Convert hex to rgb to hls
-    h, l, s = rgb_to_hls(*[int(hex_col.lstrip("#")[i : i + 2], 16) / 255 for i in (0, 2, 4)])  # noqa: E741
+    h, ll, s = rgb_to_hls(
```

Comment:
E741: ambiguous variable name -> Renamed the variable from `l` to `ll` in `gen_hex_grad` in `aeon/analysis/block_plotting.py`.